CHANGES.md

```diff
@@ -16,14 +16,43 @@ Note that it is not required to have an entry for every pull request.
 Instead, please try to add only changes that are meaningful to users who read
 this file to understand what changed.
 
-## v-next (post-v0.2.0)
+## v-next (post-v0.2.\*)
 
 - educational materials
 - new features
 - breaking changes
 - bug fixes
 - other
 
+## v0.2.1
+
+Update to `raiwidgets` package.
+
+- educational materials:
+  - add imports for `MimicExplainer` and `PFIExplainer`
+- features
+
+  - migrate cohort and dataset utilities to core-ui
+  - remove circular dependencies
+  - convert flask comms class to function
```

Comment:
This probably isn't worth mentioning, no real functionality change.

Reply:
removed "convert flask comms class to function"